hyper/pkg/proto/hyper.proto

```diff
@@ -5,8 +5,13 @@ option go_package = "github.com/isard-vdi/isard/hyper/pkg/proto";
 
 service Hyper {
     rpc DesktopStart(DesktopStartRequest) returns (DesktopStartResponse) {}
+    rpc DesktopState(DesktopStateRequest) returns (DesktopStateResponse) {}
```

Comment:
This should be below `DesktopStop` (and the message definitions below too)

Reply:
ok